cmd/tegola/cmd/cache/tile_list.go

```diff
@@ -118,7 +118,7 @@ func generateTilesForTileList(ctx context.Context, tilelist io.Reader, explicit
 			txt := scanner.Text()
 			tile, err = format.ParseTile(txt)
 			if err != nil {
-				tce.setError(fmt.Errorf("failed to parse line [%v]: %v", lineNumber, err))
+				tce.setError(fmt.Errorf("failed to parse tile (%v) from line [%v]: %v", txt, lineNumber, err))
```

Comment:
can you change the `%v` (for the err) to `%w`, the %w enables the error wrapping that was introduced in go 1.14?